packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -35,12 +35,12 @@ import {
 	checkFontFaceInstalled,
 } from './utils';
 import { toggleFont } from './utils/toggleFont';
+import setNestedValue from '../../../utils/set-nested-value';
```

Comment:
We are not duplicating the function. It already exists in the edit-site package; I just moved it to the utils folder. This function was added to the edit-site package when lodash was removed from the dependencies list.

Reply:
I know we're not duplicating the function as part of this PR, I just meant it would be good if we didn't need to duplicate it in both edit-site and core-data. But it's not really part of this PR, it can be done in a follow-up.

Reply:
> I tested fixing this by adding export * from './utils'; to the end of this [index.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/core-data/src/index.js#L88) and it worked, but I'm not sure why we're not already doing that.

As soon as a function is exported and becomes part of the public API for the package, maintaining backwards compatibility means that we're limited how it can be changed and when/if it can be removed, so it requires some thought and planning. I'm not sure that it makes sense for core-data to export all utils, but maybe just this function? Or perhaps there's a better package for such utils.